app/views/ansible_variables/index.html.erb

```diff
@@ -26,7 +26,7 @@
           variable.ansible_role.name,
           hash_for_ansible_variables_path(:search => "ansible_role = #{variable.ansible_role}")
         ) %></td>
-        <td class="ellipsis"><%= "String" %></td>
+        <td class="ellipsis"><%= variable.key_type || 'String' %></td>
```

Comment:
so now we're getting String vs string, could you please change the default to `string`?